internal/cli/command/cluster/create.go

```diff
@@ -105,8 +110,24 @@ func runCreate(banzaiCli cli.Cli, options createOptions) error {
 	}
 
 	log.Info("cluster is being created")
-	log.Infof("you can check its status with the command `banzai cluster get %q`", out["name"])
-	Out1(cluster, []string{"Id", "Name"})
+	if options.wait {
+		log.Infof("you can check its status with the command `banzai cluster get %q`", out["name"])
+		Out1(cluster, []string{"Id", "Name"})
+	} else {
+		for {
+			cluster, _, err := banzaiCli.Client().ClustersApi.GetCluster(context.Background(), orgID, cluster.Id)
+			if err != nil {
+				cli.LogAPIError("create cluster", err, out)
+			} else {
+				Out1(cluster, []string{"Id", "Name", "Status", "StatusMessage"})
+				if cluster.Status != "CREATING" {
```

Comment:
Is there a way to have the possible cluster status values defined in the pipeline client?

Reply:
Not really. Openapi supports enums, but the generated go code represents them simply by a string (the base type in fact) without consts, validators or anything for the possible values.